src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -146,7 +147,7 @@ CHE_WORKSPACE_WSAGENT__JAVA__OPTIONS           # java opts to adjust java opts o
 CHE_WORKSPACE_MAVEN__SERVER__JAVA__OPTIONS   # java opts to adjust java opts of the maven server
 ----
 
-Che admins (basically whoever has access to `che.env` or Che server environment directly) can override user-defined envs:
+Che administrators (anyone with access to the `che.env` file or the Che server environment directly) can override user-defined environment variables.
 
 ----
```

Comment:
We could use a lead-in to this.

Also, this seems like it would be better as a definition list than a monospaced block.

Reply:
Done. Also added the preceding environment variables in a definition list.